service/src/main/java/org/apache/griffin/core/metric/model/MetricValue.java

```diff
@@ -61,4 +72,38 @@ public void setTmst(Long tmst) {
     public void setValue(Map<String, Object> value) {
         this.value = value;
     }
+
+    public String getApplicationId() {
```

Comment:
Suddenly I've realized that probably more extensible way would be to have "metadata" map on metric, instead of fixed "applicationId" value. Who knows what can be useful later.

I know that applicationId is already part of the code base, but as it's not read from anywhere and we did not release that code, it can be right moment to change it. But later we'll be at point of no return.

Reply:
Ok, I think I got it. So we should add anything which would look in the json like:

    {
        ...
        "metadata": {
            "applicationId": "..."
        }
        ...
    }

Right?